src/routes.py

```diff
@@ -105,3 +105,17 @@ def create_topic() -> str | Response:
             return redirect(url_for("routes.topics"))
         return render_template("topics-create.html", form=form, current_user=current_user)
     return redirect(url_for("routes.login"))
+
+
+@bp.route("/topics/<topic_id>")
```

Comment:
I'm not sure that providing a mypy type in the signature is enough. If it works now it's a little bit weird and probably needs a little research.

And why I'm telling that is because I expected not only the changes in mypy type but in the flask route as well.
I expected it to look like in [this](https://flask.palletsprojects.com/en/1.1.x/quickstart/#variable-rules) article.